tests/quantization/test_compressed_tensors.py

```diff
@@ -767,3 +768,44 @@ def check_model(model):
 
         output = llm.generate_greedy("Hello my name is", max_tokens=4)
         assert output
+
+
+def test_compressed_tensors_moe_ignore_with_model(vllm_runner):
+    """
+    Integration test for MoE layer ignore functionality with a real model.
+
+    This test would verify that when loading a compressed-tensors quantized
+    MoE model where some MoE layers are in the ignore list, those layers
+    use UnquantizedFusedMoEMethod while non-ignored layers use the
+    quantized method.
+
+    Expected model structure:
+    - Compressed-tensors quantized MoE model (e.g., Mixtral-based)
+    - Config with ignore list containing specific MoE layers
+    - Multiple MoE layers where some are quantized and some are not
+    """
+    model_path = "nm-testing/Qwen3-30B-A3B-W4A16-first-10"
```

Comment:
This is a huge model for CI and we don't actually need to load real weights here. Can you make this for a <1B model and use `load_format="dummy"`?

Reply:
yeah i'll make a fake model for this, i don't think there's an actual MoE that's smaler than this one.